src/common/collection_column_specs/samples-ENIGMA.yml

```diff
@@ -0,0 +1,127 @@
+columns:
+  - key: coll
+    type: string
+    filter_strategy: identity
+    non_visible: true
+
+  - key: load_ver
+    type: string
+    filter_strategy: identity
+    non_visible: true
+
+  - key: _mtchsel
+    type: string
+    filter_strategy: identity
+    non_visible: true
+
+  - key: kbase_ids
+    type: string
+    filter_strategy: identity
+    non_visible: true
+    display_name: Related KBase IDs
+    category: Identifiers
+    description:
+
+  - key: genome_count
+    type: int
+    display_name: Genome Count
+    category: Statistics
+    description:
+
+  - key: kbase_sample_id
+    type: string
+    filter_strategy: identity
+    display_name: KBase Sample ID
+    category: Identifiers
+    description:
+
+  - key: kbase_display_name
+    type: string
+    filter_strategy: ngram
+    display_name: KBase Name
+    category: Identifiers
+    description:
+
+  # sample attributes currently accessible in the ENIGMA collection
+  # parsed using parse_sample_spec in parse_sample_spec.py
+  - key: enigma:collection_time
```

Comment:
If we were continuing with this project I'd say we should make a time column type but it's probably not worth the trouble

Reply:
Nothing to do here so :+1: